src/main/scala/SBTS3Resolver.scala

```diff
@@ -21,6 +23,9 @@ object SbtS3Resolver extends AutoPlugin {
     @deprecated("s3acl is now an Option. Please define it either Some(...) or None if you wish to inherit the bucket default.", "0.18.0")
     implicit def acl2Option(acl: S3ACL): Option[S3ACL] = Some(acl)
 
```

Comment:
The end of message (about bucket default) is not related to this setting. Just remove the part about None: "Please define it explicitly with Some(...).". None is default and doesn't use this conversion, so user doesn't need to see this warning if he didn't touch the setting.